debian/rules

```diff
@@ -1,114 +1,25 @@
 #!/usr/bin/make -f
-# -*- makefile -*-
-# Sample debian/rules that uses debhelper.
-# GNU copyright 1997 to 1999 by Joey Hess.
 
-# Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
-CFLAGS = -Wall -Wno-format -g
+export DEB_CFLAGS_MAINT_APPEND  = -Wall -Wno-format -Wno-format-security -g -DTAP_AND_HOLD
+export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
```

Comment:
It's better to use "VAR += ..." expression and separated "export VAR" at the end of file.
This is helpful because if someone adds new "VAR += ..." expression in the future it will allow to have good commit history for "git blame".

Reply:
Ok, sure. Both have been used through our packaging, so I just put whatever came to mind.